### ISSUE TITLE
Set-AssetOwner issue

### ISSUE THREAD

**Issue by dilpatel45** [snazy2000/SnipeitPS#68]
Hello everyone, when I try to set the asset owner I get a message stating the assigned asset and 6504 must be different. 

`Set-AssetOwner -url 'https://snipe.maderacoe.org' -apikey $apikey -id '6502' -checkout_to_type "asset" -assigned_id '6504'`

```
Invoke-SnipeitMethod :
assigned_asset
--------------
{The assigned asset and 6504 must be different.}
At C:\Program Files\WindowsPowerShell\Modules\SnipeitPS\1.0.70\Public\Set-AssetOwner.ps1:48 char:19
+         $result = Invoke-SnipeitMethod @Parameters
+                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Invoke-SnipeitMethod
```

Both the ID and the Assigned to ID are different. I'm not sure why it's still flagging it. Please advise.

**Comment by PetriAsi** [comment]
@dilpatel45 Could you provide get-asset output of asset ids '6502' and '6504'? That would show current field values before set-assetowner is run.

**Comment by PetriAsi** [comment]
This seem to be related #58

**Comment by PetriAsi** [comment]
Fixed in version v1.1.x, closing this now. Reopen if needed.
